client/blocks/reader-combined-card/index.jsx

```diff
@@ -15,13 +15,22 @@ import ReaderSiteStreamLink from 'blocks/reader-site-stream-link';
 import { siteNameFromSiteAndPost } from 'reader/utils';
 import ReaderCombinedCardPost from './post';
 
-const ReaderCombinedCard = ( { posts, site, feed, onClick, isDiscover, translate } ) => {
+const ReaderCombinedCard = ( { posts, site, feed, selectedPost, onClick, isDiscover, translate } ) => {
 	const feedId = get( feed, 'feed_ID' );
 	const siteId = get( site, 'ID' );
 	const siteIcon = get( site, 'icon.img' );
 	const feedIcon = get( feed, 'image' );
 	const streamUrl = getStreamUrl( feedId, siteId );
 	const siteName = siteNameFromSiteAndPost( site, posts[ 0 ] );
+	const isSelectedPost = post => {
```

Comment:
I think if we created a couple postKey related primitives in reader/util then some of our logic could end up similar

```jsx
postToPostKey( post ) => postKey
combinedCardToPostKeys( cc ) => [ postKeys, ... ] // or maybe combined card shouldn't have list of postIds and should have a list of postKeys instead to make things simpler.  denormalization ftw?
```

then isSelectedPost becomes:

```jsx
const isSelectedPost = post => _.isEqual( postToPostKey( post ), postToPostKey( selectedPost ) );
```